form.js

```diff
@@ -245,27 +223,16 @@ define('form', ['doc'], function($) {
 		},
 
 		/**
-		 * @param label anchor
-		 * @param interpolated message
-		 *
-		 * Usage example:
-		 * 	form.appendMessage(<INPUT_LABEL_ANCHOR>, INTERPOLATED_MESSAGE);
-		 */
-		'appendMessage': function(label, message) {
-			var $label = $(label);
-			appendMessage($label, message);
-			addEventToClearErrorMessages($label.find('input'));
-		},
-
-		/**
-		 * @param form
+		 * @param $field
 		 *
 		 * Usage example:
-		 * 	form.removeValidationErrors(<FORM_ELEMENT>);
+		 * 	form.validateField(<FIELD_ELEMENT>);
 		 */
-		'removeValidationErrors': function(form) {
-			removeValidationErrors($(form));
+		'validateField': function($field) {
```

Comment:
In the `validate` method we receive a string parameter and then transform to a doc object `var $form = toElements(form);`, to maintain consistency, should this method receive a string too? 

@tisvasconcelos